.env

```diff
@@ -4,4 +4,6 @@ REACT_APP_WEATHER_KEY=6111091c47f743c7bc1410fabffbbb88
 
 REACT_APP_MOVIE_KEY=637d69539e64f605b5d28ed0a9519e57
 
-REACT_APP_YELP_KEY=OQAY3jiadm6MIpA1yFbiZWKHSq3nRKDVUzBux8hnmHm_P8NSqnWRTJjZdIUo9bNYhaPe3q8wjPp3193GPweFht7IBUVUGXkz_ATojIKDRkzx2ouCpeX6WgwO-1WBYXYx
\ No newline at end of file
+REACT_APP_YELP_KEY=OQAY3jiadm6MIpA1yFbiZWKHSq3nRKDVUzBux8hnmHm_P8NSqnWRTJjZdIUo9bNYhaPe3q8wjPp3193GPweFht7IBUVUGXkz_ATojIKDRkzx2ouCpeX6WgwO-1WBYXYx
+
+REACT_APP_SERVER_URL=http://localhost:3001
```

Comment:
Hi there!

Feedback: .env is committed, just a heads-up here that it needs to be properly .gitignored.  👍